src/main/java/net/fabricmc/loom/util/ZipUtils.java

```diff
@@ -184,6 +184,12 @@ public static void replace(Path zip, String path, byte[] bytes) throws IOExcepti
 		}
 	}
 
+	public static void deleteIfExists(Path zip, String path) throws IOException {
+		try (FileSystemUtil.Delegate fs = FileSystemUtil.getJarFileSystem(zip, true)) {
```

Comment:
Shouldn't create be false here since you can't delete anything from a freshly created zip?

Reply:
Right, thanks!